daemon/smartcard.c

```diff
@@ -863,6 +863,33 @@ smartcard_scd_token_remove_block(container_t *container)
 	return 0;
 }
 
+int
+smartcard_remove_keyfile(smartcard_t *smartcard, const container_t *container)
+{
+	ASSERT(smartcard);
+	ASSERT(container);
+
+	int rc = -1;
+
+	char *keyfile = mem_printf("%s/%s.key", smartcard->path,
+				   uuid_string(container_get_uuid(container)));
+
+	if (NULL == keyfile) {
```

Comment:
this check can be removed, as mem_printf asserts the print was successful

Reply:
done